[2024-11-02T14:54:53.725+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dimension_batch_ingestion.ingest_account_dim manual__2024-11-02T14:54:52.280076+00:00 [queued]>
[2024-11-02T14:54:53.736+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dimension_batch_ingestion.ingest_account_dim manual__2024-11-02T14:54:52.280076+00:00 [queued]>
[2024-11-02T14:54:53.736+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T14:54:53.752+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): ingest_account_dim> on 2024-11-02 14:54:52.280076+00:00
[2024-11-02T14:54:53.757+0000] {standard_task_runner.py:57} INFO - Started process 5108 to run task
[2024-11-02T14:54:53.760+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dimension_batch_ingestion', 'ingest_account_dim', 'manual__2024-11-02T14:54:52.280076+00:00', '--job-id', '1808', '--raw', '--subdir', 'DAGS_FOLDER/loader_dag.py', '--cfg-path', '/tmp/tmpfn7vpmyx']
[2024-11-02T14:54:53.763+0000] {standard_task_runner.py:85} INFO - Job 1808: Subtask ingest_account_dim
[2024-11-02T14:54:53.819+0000] {task_command.py:415} INFO - Running <TaskInstance: dimension_batch_ingestion.ingest_account_dim manual__2024-11-02T14:54:52.280076+00:00 [running]> on host 5e812a6a7c02
[2024-11-02T14:54:53.917+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhtranquang' AIRFLOW_CTX_DAG_ID='dimension_batch_ingestion' AIRFLOW_CTX_TASK_ID='ingest_account_dim' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T14:54:52.280076+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-02T14:54:52.280076+00:00'
[2024-11-02T14:54:53.919+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-02T14:54:53.920+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -X POST -F file=@/opt/***/account_dim_large_data.csv -H "Content-Type: multipart/form-data" "http://pinot-controller:9000/ingestFromFile?tableNameWithType=account_dim_OFFLINE&batchConfigMapStr=%7B%22inputFormat%22%3A%22csv%22%2C%22recordReader.prop.delimeter%22%3A%22%7C%22%7D"']
[2024-11-02T14:54:53.929+0000] {subprocess.py:86} INFO - Output:
[2024-11-02T14:54:53.939+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-11-02T14:54:53.939+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-11-02T14:54:53.952+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100  3606  100   702  100  2904  54000   218k --:--:-- --:--:-- --:--:--  270k
[2024-11-02T14:54:53.953+0000] {subprocess.py:93} INFO - {"code":500,"error":"Caught exception when ingesting file into table: account_dim_OFFLINE. Unrecognized field \"delimeter\" (class org.apache.pinot.plugin.inputformat.csv.CSVRecordReaderConfig), not marked as ignorable (15 known properties: \"ignoreEmptyLines\", \"delimiter\", \"escapeCharacter\", \"skipUnParseableLines\", \"skipHeader\", \"quoteCharacter\", \"multiValueDelimiter\", \"multiValueDelimiterEnabled\", \"fileFormat\", \"recordSeparator\", \"commentMarker\", \"ignoreSurroundingSpaces\", \"header\", \"nullStringValue\", \"quoteMode\"])\n at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: org.apache.pinot.plugin.inputformat.csv.CSVRecordReaderConfig[\"delimeter\"])"}
[2024-11-02T14:54:53.954+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-02T14:54:53.997+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=dimension_batch_ingestion, task_id=ingest_account_dim, execution_date=20241102T145452, start_date=20241102T145453, end_date=20241102T145453
[2024-11-02T14:54:54.053+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T14:54:54.092+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
